client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/step-content.tsx

```diff
@@ -9,11 +13,47 @@ type StepContentProps = {
 	goToStep?: NavigationControls[ 'goToStep' ];
 };
 
-const StepContent = ( { siteSlug, submit, goNext, goToStep }: StepContentProps ) => (
-	<div className="launchpad__content">
-		<Sidebar siteSlug={ siteSlug } submit={ submit } goNext={ goNext } goToStep={ goToStep } />
-		<LaunchpadSitePreview siteSlug={ siteSlug } />
-	</div>
-);
+function sortByRegistrationDate( domainObjectA: ResponseDomain, domainObjectB: ResponseDomain ) {
+	const registrationDateA = domainObjectA.registrationDate;
+	const registrationDateB = domainObjectB.registrationDate;
+
+	if ( registrationDateA > registrationDateB ) {
+		return -1;
+	} else if ( registrationDateA < registrationDateB ) {
```

Comment:
Is this else if needed? Seems like if the first condition is false, we always return 1, so you could remove this else if and just go with the final return 1; statement. 

Not sure if this would be clearer, but we could (I think) simplify body of function to something like:
`return domainObjectA.registrationDate > domainObjectb.registrationDate ? 1 : -1;`